python/GangaAtlas/Lib/Athena/DQ2JobSplitter.py

```diff
@@ -599,6 +601,7 @@ def split(self,job):
             allowed_sites.append('PRAGUELCG2-RUCIOTEST_DATADISK')
 
         logger.debug('allowed_sites = %s ', allowed_sites)
+        logger.warning('allowed_sites = %s ', allowed_sites)
```

Comment:
Hi Mark, 
I needed that for debugging (without increasing debug verbosity elsewhere). I guess it can stay as `logger.debug(...)`. 
Thank you, Jarka